concent_api/middleman/views.py

```diff
@@ -0,0 +1,3 @@
+from django.shortcuts import render
```

Comment:
Files automatically created by django-admin that are not used are not necessary, you can remove them.

Reply:
Ok

Reply:
done